lib/ui/painting/multi_frame_codec.cc

```diff
@@ -145,16 +145,11 @@ sk_sp<DlImage> MultiFrameCodec::State::GetNextFrameImage(
 
 #if IMPELLER_SUPPORTS_RENDERING
   if (is_impeller_enabled_) {
-    sk_sp<DlImage> result;
-    // impeller, transfer to DlImageImpeller
-    gpu_disable_sync_switch->Execute(fml::SyncSwitch::Handlers().SetIfFalse(
-        [&result, &bitmap, &impeller_context_] {
-          result = ImageDecoderImpeller::UploadTextureToShared(
-              impeller_context_, std::make_shared<SkBitmap>(bitmap),
-              /*create_mips=*/false);
-        }));
-
-    return result;
```

Comment:
It would be nice to verify that this is fine for android opengl es as well.  This may only be an issue for iOS opengl es.

Reply:
(I think on android we don't even set the switch since it wasn't an issue)

Reply:
Android allows background GPU utilization